lib/test.gi

```diff
@@ -536,7 +543,7 @@ end);
 ## 
 ##    testOptions := rec()   : Options to pass on to Test
 ##    earlyStop := false     : Stop once one test fails
-##    showProgress := true   : Show progress
+##    showProgress := "some" : Show some progress
```

Comment:
> How does that ease maintenance? 

because otherwise there is a higher chance that someone will update documentation in `.gd` file but not comments in `.gi`

> Do I really need to rebase again to get rid of some comments? Also, which of them? 
> Why are those in 541/542 more helpful? 

I would leave 541-542 in the form
```
## TestDirectory(<files> [, <options> ])
## <files>: A directory (or filename) or list of filenames and directories
## <options>: Optional record of options
```
because they seem more stable (unlike options which may change again) and may be helpful for anyone looking at the code to quickly see the meaning of the *arguments*. The lines below may be deleted, indeed.

> Also, somebody put those comments there, so if I remove them now, that person might disagree with the change... Huh

I've approved your changes anyway, so you may wish to go ahead and merge this as it is, if you're reluctant to remove them. `TestDirectory` is by @ChrisJefferson IIRC, so maybe he would comment briefly on what he thinks...